lib/Minz/Configuration.php

```diff
@@ -198,7 +198,7 @@ public function save() {
 			return false;
 		}
 
-		// Clear PHP 5.5+ cache for include
+		// Clear PHP cache for include
 		if (function_exists('opcache_invalidate')) {
```

Comment:
Do we still need to check if the function exists?

Reply:
Some distributions do not have `opcache` by default, e.g. https://pkgs.alpinelinux.org/packages?name=php*opcache*&branch=edge&arch=x86_64 , which makes me realise that:
* We should add that do the documentation
* We should add it in the Dockerfile for Alpine
* Maybe that might explain a part of why Alpine was so much slower than Ubuntu in my last test https://github.com/FreshRSS/FreshRSS/pull/2205

Reply:
https://github.com/FreshRSS/FreshRSS/pull/2498